packages/@okta/vuepress-site/docs/guides/configure-profile-enrollment-policies/main/index.md

```diff
@@ -0,0 +1,353 @@
+---
+title: Configure profile enrollment policies
+excerpt: How to configure profile enrollment policies, self-service registration (SSR), and progressive enrollment.
+layout: Guides
+---
+
+<ApiLifecycle access="ie" /><br>
+
+> **Note:** This document is only for Okta Identity Engine. If you’re using Okta Classic Engine, see [Set up self-service registration (archived)](/docs/guides/archive-set-up-self-service-registration/main/). See [Identify your Okta solution](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-version) to determine your Okta version.
+
+This guide explains what profile enrollment policies are used for and how to add and configure them in your [Okta organization](/docs/concepts/okta-organizations/). As a result, you learn how to configure self-service registration (SSR) and progressive enrollment.
+
+---
+
+**Learning outcomes**
+
+* Know the purpose of profile enrollment policies.
+* Know the purpose of self-service registration (SSR) and progressive enrollment.
+* Add and configure profile enrollment policies, including SSR and progressive enrollment.
+
+**What you need**
+
+* [Okta Developer Edition organization](https://developer.okta.com/signup)
+* Group created in your org. See [Add group](/docs/reference/api/groups/#add-group) using the API or 
+* An application that you want to assign to an enrollment policy
```

Comment:
Most docs that I've checked mention "app", rather than "app integration". The latter would be news to me. I'll keep "app" until we collectively decide on a change.